frontend/src/modules/search/components/SearchBox.css

```diff
@@ -111,6 +111,10 @@
     content: '';
 }
 
+.search-box .missing-without-unreviewed .status:before {
+    content: '\f0eb';
+}
```

Comment:
Please make sure font-size is the same as for Unreviewed and that the icon is aligned with other Extra filters icons (probably needs margin-left of 1px).

And then also make sure the selected state icon is styled properly (font size and alignment).